### PR TITLE
Pick up changes to CRD template naming and fix excessive WG validation

### PR DESCRIPTION
**Pull request by howardjohn**
Fixes https://github.com/istio/istio/issues/52749

### CODE REVIEW

File: networking/v1alpha3/workload_group.proto [hunk 1]
```diff
@@ -109,6 +109,7 @@ message WorkloadGroup {
   // should default to `default`. The workload identities (mTLS certificates) will be bootstrapped using the
   // specified service account's token. Workload entries in this group will be in the same namespace as the
   // workload group, and inherit the labels and annotations from the above `metadata` field.
+  // +protoc-gen-crd:validation:IgnoreSubValidation:["Address is required"]
```
Comment (howardjohn) [inline networking/v1alpha3/workload_group.proto:112]: This is the meat of the change, rest is just find+replace

**Review by linsun — APPROVED** [review]

**Comment by linsun** [comment]
LG, pending tests passing.

**Review by dhawton — APPROVED** [review]

**Pull request merged**